pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md

```diff
@@ -560,4 +561,48 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them. Some of the widely used attributes are listed below. For the entire list of supported attributes, please refer to [this list](https://github.com/LivePersonInc/json-pollock/blob/master/js/schema/accessibilityWeb.json).
+
+#### Properties
+
+| Property Name    | Description                                                                 |
+| :--------------- | :-------------------------------------------------------------------------- |
+| role             | Describes the role of an element for screen readers                         |
+| tabindex         | Describes the tab navigation order of elements                              |
+| aria-label       | Describes a string value that labels an element                             |
+| aria-level       | Used together with role=heading to specify the heading level of the element |
+| aria-describedby | Reference longer content to provide description with relevant id            |
+| aria-labelledby  | Reference other elements to define accessible name                          |
+| aria-hidden      | Hide/show the element from assistive technology                             |
+
+#### Example
+
+```json
+{
+  "type": "button",
+  "title": "Test Button",
+  "click": {
+    "actions": [
+      {
+        "type": "publishText",
+        "text": "Button 1 was clicked"
+      }
+    ]
+  },
+  "accessibility": {
+      "web": {
+        "role":"heading",
+        "aria-level": "3",
+        "tabindex":"1",
+        "aria-label":"This is aria-label for button",
+        "aria-hidden":"false"
+      }
+  }
+}
+```
+
+Date Updated: 10/24/2024
```

Comment:
@pprabesh, please use a front matter entry for this information: look for “date_updated” [in the docs](https://github.com/LivePersonInc/developer-center#updating-and-creating-headers) ([example](https://github.com/LivePersonInc/developer-center/blame/master/pages/Documents/APIsandSDKs/overview.md#L13)).

Reply:
@j9t Sorry, I misunderstood earlier. Updated the front matter.